app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -389,9 +389,14 @@ class Config {
 
   Future<RepositorySlug> repoNameForBuilder(String builderName) async {
     final List<Map<String, dynamic>> builders = luciTryBuilders;
-    final String repoName = builders.firstWhere(
-        (Map<String, dynamic> builder) =>
-            builder['name'] == builderName)['repo'] as String;
+    final Map<String, dynamic> builderConfig = builders.firstWhere(
```

Comment:
Nit: some doc would be helpful for the empty case.

Reply:
> Nit: some doc would be helpful for the empty case.

Added docs.